data_processing.ipynb

```diff
@@ -772,9 +1002,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.7.5"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
 }
```

Comment:
I don't see any images being outputted. We should show a sample of a few of them in the .ipynb file.